src/test/java/it/pagopa/ecommerce/commons/v1/TransactionTestUtils.java

```diff
@@ -156,7 +158,8 @@ public static TransactionAuthorizationRequestedEvent transactionAuthorizationReq
                         PSP_CHANNEL_CODE,
                         PAYMENT_METHOD_NAME,
                         PSP_BUSINESS_NAME,
-                        AUTHORIZATION_REQUEST_ID
+                        AUTHORIZATION_REQUEST_ID,
+                        TransactionAuthorizationRequestData.PaymentGateway.valueOf(PAYMENT_GATEWAY)
```

Comment:
Add a method that taken in input the PaymentGateway enum value so that it can be used to build events for tests for each enumeration value